nvme_gw_server.py

```diff
@@ -290,8 +294,14 @@ def serve(gw_config_filename):
 
 if __name__ == "__main__":
 
-    parser = argparse.ArgumentParser()
-    parser.add_argument("gw_config_filename")
-    args = parser.parse_args()
+    parser = argparse.ArgumentParser(prog="nvme_gw", description="Manage NVMe gateways")
+    parser.add_argument(
+        "-c",
+        "--config-file",
+        default="./nvme_gw.config",
+        type=str,
+        help="config file name and location if not present in current directory",
```

Comment:
The same about the option name, default file name and option description here.

Reply:
ok